### PR TITLE
Add "TL;DR" section to README and several small fixes

### PR DESCRIPTION
**Pull request by clumsy9**
* Add "TL;DR" section with instructions to quickly run experiments to the README.
* Fix wrong command line arguments in sample commands in the "Running your own experiments" section of the README.
* Add seperate sections for both local installation and running the quickstart environment to the "System Requirements'" section of the README.
* Add small section on artifact evaluation to the "Documentation" section of the README.
* Put paths in `start_env.sh` and `run_experiments.sh` into double-quotes to support whitespaces in local repository location.
* Ensure signaure of unit test case 'test_init' in test_events.py matches the original signature.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,50 @@
 <h1 align="left">Adaptive Misuse Detection System (AMIDES)</h1>
 
-The Adaptive Misuse Detection System (AMIDES) extends conventional rule matching of SIEM systems by applying machine learning components that aim to detect attacks evading existing SIEM rules as well as otherwise undetected attack variants. It learns from SIEM rules and historical benign events and can thus estimate which SIEM rule was tried to be evaded. A brief overview of AMIDES is given in [Overview](#overview).
+> ### TL;DR
+>
+> AMIDES extends conventional rule matching of SIEM systems by machine learning components that aim to detect
+> attacks evading existing SIEM rules as well as otherwise undetected attack variants. It learns from SIEM rules
+> and historical benign events and can thus estimate which SIEM rule was tried to be evaded.  
+>
+> To run AMIDES and all the experiments of it's [publication](#documentation), execute the following commands as a *__non-root user__* on a *__linux machine__* with `docker` installed:
```
Comment (ru37z) [inline README.md]: - of it's publication -> from its paper
- linux -> Linux

File: README.md [hunk 2]
```diff
@@ -1,29 +1,50 @@
...
+## Overview
 
-For operational use, AMIDES is integrated into [Logprep](https://logprep.readthedocs.io/en/latest/user_manual/configuration/processor.html#amides), a pipeline-based log message preprocessor also written in Python. The package also contains additional scripts that help to prepare models for the operational use with Logprep. For more information on how to prepare AMIDES models for Logprep, please read [here](#preparing-models-for-logprep).
+Core of the Adaptive Misuse Detection System (AMIDES) are the misuse classification and rule attribution components. Both components deploy machine learning models. While the misuse classification component deploys a single binary classifier, the rule attribution component makes use of multiple binary classifiers that work as a multi-classifier.
```
Comment (ru37z) [inline README.md]: deploy -> employ?

File: README.md [hunk 3]
```diff
@@ -1,29 +1,50 @@
...
 
-- Physical or virtual host with a Linux-based OS
+- A physical or virtual host with a Linux-based OS
 - A minimum of 8 GB of RAM
 - At least 2 GB of HDD space
-- Python 3.10 (or newer)
-- jq
 
-The repository contains a `Dockerfile` that creates a quickstart environment for the `amides` package. For testing purposes, we highly recommend to use the quickstart environment. Building and using the environment has been tested with `docker 20.10`.
+### Local Installation
```
Comment (ru37z) [inline README.md]: Since we recommend the quickstart environment, maybe switch these two sections (i.e., quickstart first)?

File: README.md [hunk 4]
```diff
@@ -300,10 +321,12 @@ Models for the operational use of AMIDES' misuse classification and rule attribu
...
+The corresponding research paper describes AMIDES and its evaluation results in more detail:
 
 R. Uetz, M. Herzog, L. Hackländer, S. Schwarz, and M. Henze, "You Cannot Escape Me: Detecting Evasions of SIEM Rules in Enterprise Networks,"
-in *Proceedings of the 33rd USENIX Security Symposium (USENIX Security)*, 2024. [[Conference Website](https://www.usenix.org/conference/usenixsecurity24/presentation/uetz)] [[Prepublication PDF](https://www.usenix.org/system/files/sec23winter-prepub-112-uetz.pdf)]
+in *Proceedings of the 33rd USENIX Security Symposium (USENIX Security)*, 2024. [[arXiv](https://arxiv.org/pdf/2311.10197)]
+
+Artifacts play a crucial role when it comes to availabilty and reproducability of results presented in scientific papers. The code, data, and experiments described in this repository were  submiitted to the official USENIX Artifact Evaluation (AE). The AE committee awarded the submission with all of the available badges (i.e. "Artifacts Available", "Artifacts Functional", and "Results Reproduced"). More information on the artifact submission can be found in the "Artifact Appendix" of the publication.
```
Comment (ru37z) [inline README.md]: - availabilty -> availability
- reproducability -> reproducibility
- submiitted -> submitted
- USENIX -> USENIX Security Symposium's

**Review by ru37z — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (clumsy9) [reply README.md]: "employ" seems more fitting.

File: README.md (hunk 3 above)
Comment (clumsy9) [reply README.md]: That fits together better then.

File: README.md (hunk 4 above)
Comment (clumsy9) [reply README.md]: Gosh. It seems like my spell checker is not working at all...

**Review by ru37z — APPROVED** [review]

**Pull request merged**